src/Calculator/Views/Calculator.xaml

```diff
@@ -659,8 +659,7 @@
                     Content="&#xe81c;"
                     ExitDisplayModeOnAccessKeyInvoked="False"
                     TabIndex="2"
-                    Visibility="{x:Bind local:Calculator.ShouldDisplayHistoryButton(Model.IsAlwaysOnTop, Model.IsProgrammer, DockPanel.Visibility), Mode=OneWay}"
-                    IsEnabled="{x:Bind Model.IsAlwaysOnTop, Converter={StaticResource BooleanNegationConverter}, Mode=OneWay}">
```

Comment:
I am guessing the IsEnabled was originally added to disable the hotkey from being accessed when AoT on top. We should test this and add a comment to clarify instead of removing it if it turns out to be true.

Reply:
You are right, KeyboardShortcutManager doesn't check if a button is visible or not (on purpose?). The command associated to the button already checks if the dockpanel is visible or not, but I still will revert this code because not part of the original problem

Reply:
I believe it is on purpose so that some shortcuts are still accessible even when they are hidden behind the shift toggle.